packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx

```diff
@@ -39,6 +39,28 @@ So what did we install precisely?
 
 ### Android Specific installation
 
+<!-- vale off -->
+#### Update the compileSdk, targetSdk and minSdk versions
+
+In `android/build.gradle` add the following inside the `buildscript` section:
+
+```java
+buildscript {
+    ext {
+        ...
+        minSdkVersion = 24
+        compileSdkVersion = 33
+        targetSdkVersion = 33
+    }
+    ...
+}
+```
+
+:::infoINFO
```

Comment:
This should be info right?

Reply:
current info shows as success.. so overriding with INFO

Reply:
success is not a good thing to show... 

:::note is not sufficient for an important thing actually

so taking the overriding route

Reply:
you can see in the screenshot how it looks